products/BellHybrid/services/evtmgr/backlight-handler/BacklightHandler.cpp

```diff
@@ -77,6 +69,40 @@ namespace backlight
         }
     }
 
+    void Handler::handleScreenLight(Type type)
+    {
+        const auto controller = getScreenLightControl();
+        const auto timer      = getScreenLightTimer();
+
+        auto lightTime = std::chrono::seconds(0);
+
+        switch (type) {
+        case Type::Frontlight:
+            if (onDemandModeOn) {
+                lightTime = timers::screenLightTimerTimeout;
+            }
+            else {
+                lightTime = timers::screenLightTimerHoldTimeout;
+            }
+            break;
+        case Type::BedsideLamp:
+            lightTime = timers::screenLightBedsideLampTimeout;
+            break;
+        }
+
+        if (controller->isLightOn() and type == Type::Frontlight) {
```

Comment:
I'd suggest using `&&` instead of `and` for the style to be consistent throughout the whole file ;)

Reply:
Done :)